Latest build (2.24) fails when executed in a maven build on Windows

I have downgraded to version 2.22 of the plugin in the meantime.
(plugin 2.23 fails in Eclipse but not while running a mvn install')

default configuration, very simple sass file

```
$font-stack:    Helvetica, sans-serif;
$primary-color: #333;

body {
  font: 100% $font-stack;
  color: $primary-color;
}
```

Fails with this error

[plugin-2.24-fails.txt](https://github.com/GeoDienstenCentrum/sass-maven-plugin/files/1032842/plugin-2.24-fails.txt)

I have the same problem:

> Execution default of goal nl.geodienstencentrum.maven:sass-maven-plugin:2.24:update-stylesheets failed: (LoadError) no such file to load -- sass/plugin -> [Help 1]

This error occurs every time.

### Actual Behavior
Maven command:
```
mvn -e clean install
```

Logs:
```
[ERROR] Failed to execute goal nl.geodienstencentrum.maven:sass-maven-plugin:2.24:update-stylesheets (default) on project cockpitframework: Execution default of goal nl.geodienstencentrum.maven:sass-maven-plugin:2.24:update-stylesheets failed: (LoadError) no such file to load -- sass/plugin -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal nl.geodienstencentrum.maven:sass-maven-plugin:2.24:update-stylesheets (default) on project abcframework: Execution default of goal nl.geodienstencentrum.maven:sass-maven-plugin:2.24:update-stylesheets failed: (LoadError) no such file to load -- sass/plugin
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default of goal nl.geodienstencentrum.maven:sass-maven-plugin:2.24:update-stylesheets failed: (LoadError) no such file to load -- sass/plugin
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
Caused by: org.jruby.embed.EvalFailedException: (LoadError) no such file to load -- sass/plugin
	at org.jruby.embed.internal.EmbedEvalUnitImpl.run(EmbedEvalUnitImpl.java:131)
	at org.jruby.embed.ScriptingContainer.runUnit(ScriptingContainer.java:1307)
	at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1300)
	at nl.geodienstencentrum.maven.plugin.sass.AbstractSassMojo.executeSassScript(AbstractSassMojo.java:272)
	at nl.geodienstencentrum.maven.plugin.sass.compiler.UpdateStylesheetsMojo.execute(UpdateStylesheetsMojo.java:76)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- sass/plugin
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:961)
	at RUBY.require(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:130)
	at RUBY.<main>(<script>:6)
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :abcframework
```

### Environment and configuration
```
<plugin>
	<groupId>nl.geodienstencentrum.maven</groupId>
	<artifactId>sass-maven-plugin</artifactId>
	<version>2.24</version>
	<executions>
		<execution>
			<goals>
				<goal>update-stylesheets</goal>
			</goals>
		</execution>
	</executions>
	<configuration>
		<sassSourceDirectory>src/main/resources/sass</sassSourceDirectory>
		<destination>${project.build.directory}/classes/sass</destination>
		<skip>false</skip>
		<sassOptions>
			<always_update>true</always_update>
			<sourcemap>:none</sourcemap>
			<load_paths>['${basedir}/${root.project.relative.path}abc-base/acbframework/src/main/resources/sass/abc/scss']</load_paths>
		</sassOptions>
	</configuration>
</plugin>
```

```
$ mvn --version
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T18:29:23+01:00)
Maven home: D:\projects\platform\suite\build-tools\apache-maven\bin\..
Java version: 1.8.0_112, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0_112\jre
Default locale: en_US, platform encoding: Cp1250
OS name: "windows 10", version: "10.0", arch: "amd64", family: "dos"
```

```
$ java -version
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)
```

```
Edition: Windows 10 Enterprise
Version: 1511
OS Build: 10586.873
System type: 64-bit operating system, x64-based processor
```

--- a/pom.xml
+++ b/pom.xml
@@ -142,9 +142,9 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<compass.version>1.0.3</compass.version>
 		<sass.version>3.4.24</sass.version>
-		<jruby.version>9.1.10.0</jruby.version>
+		<jruby.version>9.1.12.0</jruby.version>
 		<maven-libs.version>3.5.0</maven-libs.version>
-		<scss-lint.version>0.53.0</scss-lint.version>
+		<scss-lint.version>0.54.0</scss-lint.version>
 		<bourbon.version>4.3.4</bourbon.version>
 		<maven.version />
 		<debug>true</debug>
@@ -357,7 +357,7 @@
 				<plugin>
 					<groupId>org.codehaus.mojo</groupId>
 					<artifactId>versions-maven-plugin</artifactId>
-					<version>2.3</version>
+					<version>2.4</version>
 				</plugin>
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>

--- a/src/site/apt/development.apt.vm
+++ b/src/site/apt/development.apt.vm
@@ -66,13 +66,13 @@ mvn -e verify -B
   run:
 
 +----+
-mvn -U org.codehaus.mojo:versions-maven-plugin:2.3:display-dependency-updates
+mvn -U org.codehaus.mojo:versions-maven-plugin:2.4:display-dependency-updates
 +----+
 
   to check for plugin updates and run:
 
 +----+
-mvn -U org.codehaus.mojo:versions-maven-plugin:2.3:display-plugin-updates
+mvn -U org.codehaus.mojo:versions-maven-plugin:2.4:display-plugin-updates
 +----+
 
   to check for plugin updates.

--- a/src/site/apt/releasenotes.apt.vm
+++ b/src/site/apt/releasenotes.apt.vm
@@ -36,11 +36,14 @@ ${project.name} ${project.version} Release Notes
 
 ~~   []
 
-~~** Enhancements and Fixes
+** Enhancements and Fixes
 
-~~   *
+  * Update JRuby to 9.1.12.0
 
-~~   []
+  * Update SCSS-Lint to version 0.54.0, please check
+    the {{{https://github.com/brigade/scss-lint/blob/master/CHANGELOG.md}SCSS-Lint changelog}}
+
+  []
 
 * 2.24 Release Notes
 

